CONTRIBUTING.rst

```diff
@@ -174,9 +178,13 @@ release process for this package is:
 
 1. Update ``version`` in ``packages/pytest-taskgraph/pyproject.toml``
 2. Commit and land the changes with a commit message like "chore: bump pytest-taskgraph <version>"
-3. Create a release in Github pointing to the above commit. Be sure to also
-   create a new tag of the form ``pytest-taskgraph-v<version>``.
-4. Wait for the ``pypi-publish`` Github workflow and ``push-image-decision`` task to finish.
+3. Draft a release in Github pointing to the above commit.
+
+   a. Create a new tag of the form ``pytest-taskgraph-vX.Y.Z``
```

Comment:
Bit weird to have `vX.Y.Z` on this tag but not on the other. Don't have a very strong opinion on this though